internal/sidecarlogresults/sidecarlogresults.go

```diff
@@ -17,15 +17,26 @@ limitations under the License.
 package sidecarlogresults
```

Comment:
I did have it in pkg/internal before but then I was unable to import it in cms/sidecarlogresults/main.go which is why I had to move it here. 

the reason they are two different packages is because in my final PR I will be doing webhook validation to ensure that a user does not create a sidecar with the same name as the one that is injected. I run into cyclic imports there where sidecarlogresults would import v1beta1 and vice versa (to get access to the reservedSidecarName.

Reply:
moving it to pkg/apis/pipeline again :)

Reply:
> the reason they are two different packages is because in my final PR I will be doing webhook validation to ensure that a user does not create a sidecar with the same name as the one that is injected.

I think putting reserved names (such as the sidecar name) in pkg/apis/pipeline makes sense. Does that result in import cycles as well?

Reply:
No, Moving it to pkg/apis/pipeline does not result in any import cycle issue. Moved it there so I think we're good here.